playgrounds/skeleton-react/src/app/composed/accordion/page.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
```

Comment:
The playgrounds are a great starting place while we update and refine, but just want to make it clear we'll need to document in the main website docs too. I figure we create a new navigation category like:

- `/components`
- `/components-compound`

Then in the visible Navigation component, set the titles as:

```
Functional Components

...

Compound Components (Beta)

...
```

This will be temporary as don't think we can fit "Functional Components (Beta)" as is.